src/pages/permissions/actions/DeleteIdentityBtn.tsx

```diff
@@ -33,8 +33,8 @@ const DeleteIdentityBtn: FC<Props> = ({ identity }) => {
         });
         toastNotify.success(
           <>
-            Identity <ResourceLabel type={"idp-group"} value={identity.name} />{" "}
-            deleted.
+            {identity.authentication_method.toUpperCase()} Identity{" "}
+            <ResourceLabel type={"idp-group"} value={identity.name} /> deleted.
```

Comment:
Why add the auth method of the deleted identity to the message here? I think without the method, it is shorter to read and easier to understand.

Reply:
The attached figma file only differentiates between OIDC and TLS identities, not between OIDC, TLS (Fine-grained) and TLS Legacy, which I feel may be more appropriate for this UI. That being said, I implemented your suggestion and I think it is a nice step in the right direction. Let me know your thoughts :)

Reply:
TLS legacy and TLS fine-grained are both certificates, so they can be represented by the same icon. Furthermore, these messages will never show a legacy TLS identity since you can't delete them.

Unresolving this comment for now, I think we should handle the following issues:
1. We should remove the auth method from the message.
2. The resource label type should be either `oidc-identity` or `certificate` depending on identity type.